nft_ingester/src/program_transformers/token_metadata/v1_asset.rs

```diff
@@ -297,6 +299,35 @@ pub async fn save_v1_asset<T: ConnectionTrait + TransactionTrait>(
             txn.execute(query)
                 .await
                 .map_err(|db_err| IngesterError::AssetIndexError(db_err.to_string()))?;
+        } else {
+            let model = asset_grouping::ActiveModel {
+                asset_id: Set(id.to_vec()),
+                group_key: Set("collection".to_string()),
+                group_value: Set(None),
+                seq: Set(0),
+                slot_updated: Set(slot_i),
+                ..Default::default()
+            };
+            let mut query = asset_grouping::Entity::insert(model)
+                .on_conflict(
+                    OnConflict::columns([
+                        asset_grouping::Column::AssetId,
+                        asset_grouping::Column::GroupKey,
+                    ])
+                    .update_columns([
+                        asset_grouping::Column::GroupValue,
+                        asset_grouping::Column::SlotUpdated,
+                    ])
+                    .to_owned(),
+                )
+                .build(DbBackend::Postgres);
+            query.sql = format!(
+                "{} WHERE excluded.slot_updated > asset_grouping.slot_updated AND excluded.seq >= asset_grouping.seq",
```

Comment:
Here I've wrote `AND excluded.seq >= asset_grouping.seq` to be consistent with previous ones. But this is not required.